Sources/Rendering/Core/VolumeMapper/index.js

```diff
@@ -74,24 +76,29 @@ function vtkVolumeMapper(publicAPI, model) {
   };
 
   publicAPI.setGlobalIlluminationReach = (gl) => {
-    model.globalIlluminationReach = vtkMath.clampValue(gl, 0.0, 1.0);
-    publicAPI.modified();
+    superClass.setGlobalIlluminationReach(vtkMath.clampValue(gl, 0.0, 1.0));
   };
 
   publicAPI.setVolumetricScatteringBlending = (vsb) => {
-    model.volumetricScatteringBlending = vtkMath.clampValue(vsb, 0.0, 1.0);
-
-    publicAPI.modified();
+    superClass.setVolumetricScatteringBlending(
```

Comment:
Best to return the return value of the inner `set` calls, since setters return a boolean whether the value changed or not.